### PR TITLE
adding compat data for svg attribute data-* and svg attribute href

### PR DESCRIPTION
**Pull request by ArmaanSethi**
ran into "https://developer.mozilla.org/docs/Web/SVG/Attribute/href" and saw it needed to be updated. 
Followed both "https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables" and "https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md", but was not sure what to name the file.

### CODE REVIEW

**Comment by Elchi3** [comment]
Hi, thanks for contributing to mdn-browser-compat-data.

"`data-*`" can't be an identifier, because the `*` isn't allowed in identifiers. You can add it in a `description` though. I'll let Jeremie review, as I'm unsure if this is intended to be added to BCD this way, or if the docs are being revamped as part of his [SVG project](https://discourse.mozilla.org/t/time-to-lift-up-the-svg-documentation-to-the-next-level/25529).

File: svg/attributes/data.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
+  "svg": {
+    "attributes": {
+      "href": {
```
Comment (JeremiePat) [inline svg/attributes/data.json]: You should remove that object as href data are already contributed in each individual element (it worth checking if the data are up to date there)

File: svg/attributes/data.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "data-*": {
```
Comment (JeremiePat) [inline svg/attributes/data.json]: rename to `data`

File: svg/attributes/data.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Attribute/data-*",
```
Comment (JeremiePat) [inline svg/attributes/data.json]: Add a `description` field with the string: `<code>data-*</code>`

**Comment by JeremiePat** [comment]
Hi :)

Thanks for the help on SVG :)

Two things here:

1. Yes this is the right place to add compat data for `data-*`. I would add the same concern as @Elchi3 : Just change the identifier for `data` and add a `description` field to have the human friendly string `data-*`
2. On the other hand, `href` is a different beast. It is an attribute that should be contributed at each individual element level (and it's already the case). What is missing currently is a kumascript macro able to aggregate the data from each elements in a single place to populate the MDN page for the attribute. I'm unsure if we have such a macro but I remember @wbamberg has done some testing for doing this for the web extension documentation.

For details, see my review on the code :)

**Comment by JeremiePat** [comment]
After careful thinking I think it would maybe more appropriate to move `data-*` into [svg/attributes/core.json](https://github.com/mdn/browser-compat-data/blob/master/svg/attributes/core.json)

**Comment by Elchi3** [comment]
I agree with Jeremie on having `data-*` in [svg/attributes/core.json](https://github.com/mdn/browser-compat-data/blob/master/svg/attributes/core.json). I think we should open a new PR. No comment from the original contributor in over 2 months. Closing.

**Pull request closed without merging**